Vagrantfile

```diff
@@ -23,6 +23,19 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.provision "file", source: File.join(Dir.home, ".gitconfig"), destination: ".gitconfig"
   end
 
+  # Use the boto configuration from the host in the VM, if it is in the
+  # expected location
+  if File.exists?(File.join(Dir.home, ".boto"))
+    config.vm.provision "file", source: File.join(Dir.home, ".boto"), destination: ".boto"
+  end
+
+  if ENV.has_key?('VERSION')
```

Comment:
Separately, copying over environment variables is super-brittle-magic and not a good idea (what if the user decides they prefer ZSH and immediately switch to that?).